etc/perf.yml

```diff
@@ -641,6 +672,14 @@ tasks:
         variant1 : "linux-mmap-standalone"
         variant2 : "linux-mmap-repl"
     - func: "analyze"
+- name: benchmark
```

Comment:
Lets rename this cppMicroBenchmarks. I know it's silly, but that will cause it to sort immediately after compile which should the order to be nicer when viewed on evergreen.